docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -37,21 +41,22 @@ GUI clients need to present a certificate to the Teleport Database Service and
 check the database server certificate issued by a Teleport-protected database.
 There are three ways to do this:
 
-- **Authenticated tunnel (recommended):** The local proxy handles loading the
-  Teleport user's client certificate and checking the certificate issued by the
-  Teleport-protected database. The connection remains authenticated, and clients
-  only need to forward database protocol messages to it.
-- **Unauthenticated local proxy:** The local GUI client needs to load the
-  Teleport user's certificate and check database certificates. This requires
-  configuring the client to access TLS credentials, including the user
-  certificate and private key, and a certificate issued by the database's
-  certificate authority. 
+- **Authenticated tunnel (recommended):** `tsh` or Teleport Connect starts a
+  local proxy, which handles loading the Teleport user's client certificate and
+  checking the certificate issued by the Teleport-protected database. The
+  connection remains authenticated, and clients only need to forward database
+  protocol messages to it.
+- **Unauthenticated local proxy:** `tsh` or Teleport Connect starts a local
+  proxy for database traffic, including authentication from clients. The GUI
+  client needs to load the Teleport user's certificate and check database
+  certificates. This requires configuring the client to access TLS credentials,
+  including the user certificate and private key, and a certificate issued by
+  the database's certificate authority. 
```

Comment:
"check database certificates" in this context is confusing for me because I don't think there's any checking involved on the side of the GUI client. It rather needs to provide these certificates for the connection.

I think it could be simplified to something like this:

> tsh starts a local proxy which routes connections to the Teleport-protected database, but the proxy itself doesn't handle authentication. This requires configuring the database GUI client to provide the user certificate and the private key, as well as a CA certificate, all available under the paths reported by `tsh proxy db`.

Reply:
Thanks! Added this wording with some minor edits to tighten it up and avoid mentioning `tsh proxy db` before we introduce the command.